### PR TITLE
added --file option to mapper command

### PR DESCRIPTION
**Pull request by YehyaHadi**
https://sjsu.instructure.com/courses/1375041/discussion_topics/4088025

### CODE REVIEW

**Comment by spencer-enriquez** [comment]
Looks good!

**Review by StevenvL — APPROVED** [review]
Looks great!

**Review by spencer-enriquez — APPROVED** [review]
Looks good to me!

File: utility/mac_mapper.py [hunk 1]
```diff
@@ -60,6 +63,10 @@ def mac_mapper():
     with open('mac_mapper.txt', "w") as f:
         f.write(text)
 
+    with open(file, "w") as f:
```
Comment (breed) [inline utility/mac_mapper.py:66]: i thought file was optional. is that not the case?

File: utility/mac_mapper.py [hunk 2]
```diff
@@ -45,7 +46,9 @@ def mac_mapper():
                     element = str(varBind)
                     element = element.replace("SNMPv2-SMI::mib-2.17.4.3.1.2.", "").replace(" = ", ";")
                     splitArr = element.split(";")
-                    data.append(host + ',' + element.replace(splitArr[0],decToHexAddress(splitArr[0])))
+                    mac_address = element.replace(splitArr[0],decToHexAddress(splitArr[0]))
+                    mac_addresses.append(mac_address)
+                    data.append(host + ',' + mac_address)
```
Comment (breed) [inline utility/mac_mapper.py:51]: does mac_address still have the port? is the interface also there?

**Pull request closed without merging**